apps/filecoin-site/src/app/[locale]/case-studies/utils/generateCaseStudyArticleStructuredData.ts

```diff
@@ -0,0 +1,50 @@
+import type { Article } from 'schema-dts'
+
+import type { ArticleGraph } from '@filecoin-foundation/ui/StructuredDataScript'
+import { SCHEMA_CONTEXT_URL } from '@filecoin-foundation/utils/constants/structuredDataConstants'
+
+import { PATHS, type NextRouteWithoutLocale } from '@/constants/paths'
+import { BASE_URL } from '@/constants/siteMetadata'
+import { STRUCTURED_DATA_IDS } from '@/constants/structuredDataConstants'
+
+import { generateBreadcrumbList } from '@/utils/generateBreadcrumbsList'
+
+type GenerateCaseStudyArticleStructuredDataProps = {
+  path: NextRouteWithoutLocale
+  headline: string
+  description: Article['description']
+  image?: Article['image']
+}
+
+export function generateCaseStudyArticleStructuredData({
+  path,
+  headline,
+  description,
+  image,
+}: GenerateCaseStudyArticleStructuredDataProps): ArticleGraph {
+  const url = `${BASE_URL}${path}`
+
+  const caseStudyArticle: Article = {
+    '@type': 'Article',
+    headline,
+    description,
+    author: { '@id': STRUCTURED_DATA_IDS.ORGANIZATION },
+    publisher: { '@id': STRUCTURED_DATA_IDS.ORGANIZATION },
+    mainEntityOfPage: { '@id': url },
+    ...(image && { image }),
+  }
```

Comment:
Is this right? The author is `Thing`. Should it be `Organization`..?

<img width="1688" height="792" alt="Image" src="https://github.com/user-attachments/assets/99af7790-b975-4947-840f-260292a66862" />